documentation/assemblies/assembly-iot-getting-started.adoc

```diff
@@ -0,0 +1,32 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-iot-guide.adoc
+
+:context: {context}-iot
+
+[id='iot-getting-started-{context}]
+== Getting started
+
+The IoT guide provides resources on how to set up and manage {ProductName} IoT features.
+
```

Comment:
Suggest moving the IoT concept module here (include::../modules/con-iot-concepts.adoc[leveloffset=+1])